features/user_can_post.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Create articles
+  As a visitor
+  In order to discuss plants with others 
+  I would like to be able to post to the forum
```

Comment:
In  PT the user story is  
```
As a user,
In order to communicate with other users on Florum,
I would like to be able to generate a post
```
Here, you have altered it. Why?

Reply:
We will change this PT user story to visitor and add a new user story where only user can create posts